st2common/bin/st2ctl

```diff
@@ -57,16 +57,27 @@ function validate_in_components() {
 
 function st2start() {
   for COM in ${COMPONENTS}; do
-    service ${COM} start
+    service_manager ${COM} start
   done
 }
 
 function st2stop() {
   for COM in ${COMPONENTS}; do
-    service ${COM} stop
+    service_manager ${COM} stop
   done
 }
 
+function service_manager() {
+  if command -v systemctl 2>/dev/null; then
+    systemctl "${2}" "${1}"
```

Comment:
> Also I check the same way,
https://github.com/StackStorm/st2-packages/blob/master/packages/st2actions/bin/runners.sh#L17

And you're running `systemctl` binary: https://github.com/StackStorm/st2-packages/blob/master/packages/st2actions/bin/runners.sh#L20


I rely on existence of the toolset, rather then on init system, because our case is really simple: if `systemctl` exists - we want to use it, otherwise `service`.
I can put `service` before `systemctl`, to be sure. That'll cover us, just `service` will redirect to `systemctl` in CentOS7.

----

Also `service` already worked without issues before: https://github.com/StackStorm/st2/blob/master/tools/st2ctl#L152-L157 in AIO installer. See my case when `service` exists by default for compatibility in `CentOS7`: https://github.com/StackStorm/st2/pull/2367#discussion_r49769025